packages/hub/src/lib/parse-safetensors-metadata.ts

```diff
@@ -139,12 +139,13 @@ async function parseShardedIndex(
 		throw new SafetensorParseError(`Failed to parse file ${path}: not a valid JSON.`);
 	}
 
+	const pathPrefix = path.substr(0, path.lastIndexOf("/") + 1);
```

Comment:
`substr` is deprecated for strings: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substr

(`.slice` is my goto usually)

Reply:
https://github.com/huggingface/huggingface.js/pull/598

Reply:
it's still in the LLMs though =)